pkgs/applications/editors/codeblocks/default.nix

```diff
@@ -119,7 +119,11 @@ stdenv.mkDerivation rec {
   preConfigure = "substituteInPlace ./configure --replace /usr/bin/file ${file}/bin/file";
   postConfigure = lib.optionalString stdenv.isLinux "substituteInPlace libtool --replace ldconfig ${stdenv.cc.libc.bin}/bin/ldconfig";
   configureFlags = [ "--enable-pch=no" ] ++ lib.optionals contribPlugins [
-    ("--with-contrib-plugins" + lib.optionalString stdenv.isDarwin "=all,-FileManager,-NassiShneiderman")
+    (
+      "--with-contrib-plugins"
```

Comment:
`all` is the implicit default so this will try to build filemanager (depending on gamin) on `!linux && !darwin`.